OfferZen-Community/developer-mentoring
# style: add proxima nova font globally

### Gif Tax:
![](https://media0.giphy.com/media/h6AMD4GXFxO2k/giphy.gif?cid=ecf05e47cebpy1ljjlmlfyxw0jn05ltqepl1gly7usof7qvv&rid=giphy.gif&ct=g)
## Issue

-  #54 

## Steps to Test
- Run on local - `npm start`
- Visit page - `localhost:3000` 
- Confirm font has changed

## Screenshots
<img width="1415" alt="Screenshot 2022-06-08 at 20 05 42" src="https://user-images.githubusercontent.com/18310217/172686677-490b35c2-806d-47f7-92c1-cddc647d23c5.png">
